### PR TITLE
Updated docs with new SDK version details

### PR DESCRIPTION
**Pull request by tusharojha**
Updates include:
- Quick Reference guide, checked and updated.
- Intro & Getting started related docs

Working on how to guide, will create separate PR for it.

### CODE REVIEW

**Comment by tusharojha** [comment]
@olehmell @Yung-Beef please review.

File: docs/develop/how-to-guides/ConnectToSubsocial.md [hunk 1]
```diff
@@ -115,7 +114,7 @@ The above code fetches all the postIds of the space. Now, we can use these Ids t
 ```typescript
   // Fetching the first post from the list of postIds.
   // Use substrateApi to run Subsocial's pallet functions to get data about posts and content on the chain.
-  const substrateApi = await flatApi.subsocial.substrate.api
+  const substrateApi = await api.blockchain.api
```
Comment (olehmell) [inline docs/develop/how-to-guides/ConnectToSubsocial.md:117]:
```suggestion
  const substrateApi = await api.substrateApi
```

**Review by olehmell — CHANGES_REQUESTED** [review]

File: docs/develop/getting-started/APITypes.md [hunk 2]
```diff
@@ -4,26 +4,25 @@ title: API Types
 displayed_sidebar: developSidebar
 ---
 
-## Flat API
+## General API
 
-Flat API creates a new connection to the Flat Subsocial API (with a wrapper which returns flat structs).
-Thus, using the Flat, we abstract from the blockchain and can use the usual JavaScript types.
+Subsocial API creates a new connection to the blcokchain and provides a list of methods that help in fetching and writing data on substrate pallets. It uses proper types for javascript and are easy for developing most of the social Dapps.
```
Comment (Yung-Beef) [inline docs/develop/getting-started/APITypes.md:9]:
```suggestion
Subsocial API creates a new connection to the blockchain and provides a list of methods that help in fetching and writing data on substrate pallets. It uses proper types for javascript and is easy for developing most social Dapps.
```

File: docs/develop/getting-started/APITypes.md [hunk 3]
```diff
@@ -4,26 +4,25 @@ title: API Types
...
 ```
 
 >🆃 NewSubsocialApiProps: Omit<[*SubsocialApiProps*](https://docs.subsocial.network/js-docs/js-sdk/modules/api.html#subsocialapiprops), 'substrateApi'> & { substrateNodeUrl: *string*, substrateApi?: *ApiPromise* }
 
 
-## SubsocialApi
+## Core Subsocial API
 
-SubsocialApi creates a new connection to the Subsocial API (includes Substrate and IPFS connections).
+Core SubsocialApi provides you with detailed methods for interacting with Subsocial blockchain.
```
Comment (Yung-Beef) [inline docs/develop/getting-started/APITypes.md:20]:
```suggestion
The core SubsocialApi provides you with detailed methods for interacting with the Subsocial blockchain. 
```

File: docs/develop/how-to-guides/ConnectToSubsocial.md [hunk 4]
```diff
@@ -125,7 +124,7 @@ The above code fetches all the postIds of the space. Now, we can use these Ids t
 ```
 
 Lets understand the example. First we connected to Subsocial using the newFlatSubsocialApi function that
-needs the Substrate node URL, offchain URL, and IPFS node URL. After initialization, flatApi can be
+needs the Substrate node URL, offchain URL, and IPFS node URL. After initialization, api can be
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/ConnectToSubsocial.md:127]:
```suggestion
needs the Substrate node URL, offchain URL, and IPFS node URL. After initialization, the API can be
```

File: docs/develop/getting-started/APITypes.md [hunk 5]
```diff
@@ -4,26 +4,25 @@ title: API Types
...
+## Core Subsocial API
 
-SubsocialApi creates a new connection to the Subsocial API (includes Substrate and IPFS connections).
+Core SubsocialApi provides you with detailed methods for interacting with Subsocial blockchain. 
 
 Using this class, you can get all the data of posts, spaces and profiles from IPFS and the blockchain's storage with appropriate methods.
 
 ```
-SubsocialApi(props: SubsocialApiProps): SubsocialApi
+You can access it from the SubsocialApi object like this: `api.subsocial`.
```
[the remark this answers is not in the document]
Comment (tusharojha) [reply docs/develop/getting-started/APITypes.md:25]: base subsocial API.

**Pull request merged**